api/types/access_request.go

```diff
@@ -826,3 +827,25 @@ func NewAccessRequestAllowedPromotions(promotions []*AccessRequestAllowedPromoti
 		Promotions: promotions,
 	}
 }
+
+// ValidateAssumeStartTime returns error if start time is in an invalid range.
+func ValidateAssumeStartTime(assumeStartTime time.Time, accessExpiry time.Time, creationTime time.Time) error {
+	// Guard against requesting a start time before the request creation time.
+	if assumeStartTime.Before(creationTime) {
+		return trace.BadParameter("assume start time has to be after: %q", creationTime.Format(time.RFC3339))
```

Comment:
```suggestion
		return trace.BadParameter("assume start time has to be after %v", creationTime.Format(time.RFC3339))
```

I don't know that quoting the creation time is necessary. The quote helps if we want to be able to identify an empty string, but formatting a time will always product a non-empty string.

I also don't think `:` is necessary - just make it read like a normal sentence.

(don't forget to update your tests if you decide to accept these suggestions)